### PR TITLE
Added REV and FIRST Team Support Contacts

### PR DESCRIPTION
**Pull request by marshallmassengill**
Added support contact emails for REV and FIRST.

### CODE REVIEW

File: source/docs/software/support/support-resources.rst [hunk 1]
```diff
@@ -29,6 +29,16 @@ CTRE Support
 
 Support for Cross The Road Electronics components (Pneumatics Control Module, Power Distribution Panel, Talon SRX, and Voltage Regulator Module) is provided via the e-mail address support@crosstheroadelectronics.com
 
+REV Robotics Support
+------------
```
Comment (Daltz333) [inline source/docs/software/support/support-resources.rst:33]: Gotta extend this to be the num characters of the title.

File: source/docs/software/support/support-resources.rst [hunk 2]
```diff
@@ -29,6 +29,16 @@ CTRE Support
...
+
+Support for REV Robotics components (SPARK MAX, Sensors, upcoming new control system components) is provided via phone at 844-255-2267 or via the e-mail address support@revrobotics.com
```
Comment (Daltz333) [inline source/docs/software/support/support-resources.rst:35]:
```suggestion
Support for REV Robotics components (SPARK MAX, Sensors, upcoming new control system components) is provided via phone at 844-255-2267 or via the email address ` support@revrobotics.com <mailto:support@revrobotics.com`
```

File: source/docs/software/support/support-resources.rst (hunk 1 above)
Comment (marshallmassengill) [reply source/docs/software/support/support-resources.rst]: Oops - missed that.  Trying to do 20 things at once and something broke.  Sadly it was this build.

Comment (Daltz333) [reply source/docs/software/support/support-resources.rst]: No worries. I'll push the link changes for ya when I get off work ;)

Comment (marshallmassengill) [reply source/docs/software/support/support-resources.rst]: Hopefully it's fixed now but if not then let me know.  I'm having to do this from a browser at the moment and I need to get setup for editing this locally.

Comment (Daltz333) [reply source/docs/software/support/support-resources.rst]: Yep. I'm more thinking of my other comment. We can actually turn these into clickable links that open a users phone or email client with "mailto:" and "tel" links

Comment (marshallmassengill) [reply source/docs/software/support/support-resources.rst]: Gotcha.  Yeah, if you can make the REV one pop up with a subject line of "Hi Greg" that would be great. ;)

**Review by sciencewhiz — CHANGES_REQUESTED** [review]
The PR should be targeted to main not stable.

**Comment by Kevin-OConnor** [comment]
The FRC Team Support e-mail is not the best place to go for technical assistance. The [_FIRST_ Forums ](https://forums.firstinspires.org/) is where we direct people for technical help or feedback.

**Comment by marshallmassengill** [comment]
> The FRC Team Support e-mail is not the best place to go for technical assistance. The [_FIRST_ Forums ](https://forums.firstinspires.org/) is where we direct people for technical help or feedback.

Ok.  I'll remove the FIRST email.  I'm working on tracking down what to do for NI.  I'll try to get an update pushed later today resolving the branch issue and this one at a minimum.

**Comment by mack-6326** [comment]
As a side note: one of the items that is really useful for any support tickets for REV products is using the "Send Diagnostic Data to REV" feature in the REV Hardware Client. Think this is above/beyond what is needed here but likely something to revisit as the REV Control System components are released.

**Comment by Kevin-OConnor** [comment]
NI should be directed to the NI FRC Support Forum here: https://forums.ni.com/t5/Support/ct-p/Support?profile.language=en

**Comment by marshallmassengill** [comment]
Reopening against master instead of stable.

**Comment by sciencewhiz** [comment]
FYI, You can retarget a PR through the GitHub GUI https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/changing-the-base-branch-of-a-pull-request

**Comment by marshallmassengill** [comment]
I did actually try that at first but it was not populating the drop down.  This should still work to accomplish the tactical goal though strategically, I agree that this could have been done much more easily.

**Pull request closed without merging**